src/Tizen.NUI/src/public/BaseComponents/Style/TextFieldStyle.cs

```diff
@@ -438,15 +438,15 @@ public class TextFieldStyle : ViewStyle
         });
         /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public static readonly BindableProperty PixelSizeProperty = BindableProperty.Create(nameof(PixelSize), typeof(float?), typeof(TextFieldStyle), null, propertyChanged: (bindable, oldValue, newValue) =>
+        public static readonly BindableProperty PixelSizeSelectorProperty = BindableProperty.Create("PixelSizeSelector", typeof(float?), typeof(TextFieldStyle), null, propertyChanged: (bindable, oldValue, newValue) =>
```

Comment:
I think it would be better to use 'nameof(PixelSizeSelector)'

Reply:
You are right. But unfortunately, there is no `PixelSizeSelector` in `TextField`.

Reply:
Done